src/utilcode/debug.cpp

```diff
@@ -622,8 +622,6 @@ unsigned DbgGetEXETimeStamp()
 // // //  Don't call this with a K larger than 159.
 // // //
 
-#if !defined(NO_CRT)
```

Comment:
I am a bit confused - the NO_CRT was defined before and so the code under this `#if` was excluded. Now we have got rid of the `#if`, but we include the previously excluded stuff here.